test/new-e2e/tests/agent-platform/platforms/platforms.json

```
@@ -49,20 +51,20 @@
     "centos": {
         "x86_64": {
             "centos-610": "ami-0506f01ccb6dddeda",
-            "centos-79": "ami-0db0376aef6830be3",
+            "centos-79": "ami-036de472bb001ae9c",
             "rhel-86": "ami-031eff1ae75bb87e4",
             "rocky-92": "ami-08f362c39d03a4eb5",
             "rhel-86-fips": "ami-0d0fb96b595c56e03"
         },
         "arm64": {
-            "centos-79": "ami-0144a5a84f5699847",
+            "centos-79": "ami-0cb7a00afccf30559",
             "rhel-86": "ami-0238411fb452f8275"
         }
     },
     "suse": {
         "x86_64": {
-            "sles-12": "ami-08d21b039336d9351",
-            "sles-15": "ami-08f3662e2d5b3989a"
+            "sles-12": "ami-058c3f61e5f37c43f",
```

```suggestion
            "sles-12": "ami-0b0597153739840c4",
```
from https://github.com/DataDog/datadog-agent/pull/33292